daemon/daemon_windows.go

```diff
@@ -48,7 +48,7 @@ func getPluginExecRoot(root string) string {
 	return filepath.Join(root, "plugins")
 }
 
-func getBlkioWeightDevices(config *containertypes.HostConfig) ([]blkiodev.WeightDevice, error) {
+func getBlkioWeightDevices(config containertypes.Resources) ([]specs.LinuxWeightDevice, error) {
```

Comment:
Looks like this function is not used anywhere; the only place `getBlkioWeightDevices()` is called is in `oci_linux.go`, which is not compiled on Windows; https://github.com/moby/moby/blob/f70c715be01f5d5f4a8be4ab1b2c6e1464b478a7/daemon/oci_linux.go#L37-L40

Reply:
^ @dqminh